dockerfiles/docker-myeloseq/QC_metrics.pl

```diff
@@ -151,6 +150,8 @@
         push @values, $value;
     }
 
+    push @values, $data->{QC}->{'QC STATUS'}
```

Comment:
The new metrics "QC STATUS" can be added to %group4 as "QC_STATUS  => 'QC STATUS'" so line153 and extra addition in line 64 are not needed.